packages/imperative/src/console/src/Console.ts

```diff
@@ -175,7 +175,7 @@ export class Console implements IConsole {
         }
         let adjustedMessage = message;
         if (this.prefix) {
-            adjustedMessage = this.buildPrefix("FATAL") + message;
+            adjustedMessage = Console.buildPrefix("FATAL") + message;
```

Comment:
Please make sure to update the tests after making this private method static.

https://github.com/zowe/zowe-cli/pull/2478/files#file-packages-imperative-src-console-__tests__-console-unit-test-ts-L86

I believe you still need to do `(Console as any).buildPrefix` in the test file.

- `packages/imperative/src/console/__tests__/Console.unit.test.ts`

Reply:
Yes, I have used `Console.buildPrefix` for the test :relaxed: